libs/blocks/marketo/marketo.css

```diff
@@ -197,6 +201,12 @@
 
 .dialog-modal .marketo {
   margin: 0;
+  background-color: #FAFAFA;
```

Comment:
nit: when using a color more than once I like to put it into a variable to make it easier to update the color later on.